test/liquidation/liquidation.test.js

```diff
@@ -95,4 +88,31 @@ describe("Integration -- liquidation happy path", () => {
       )
     })
   })
+
+  describe("when deposit has changed state between auction creation and auction offer taken", () => {
+    let auction
+    beforeEach(async () => {
+      auction = await prepareAuction()
+      // simulate deposit state change outside Coverge Pools
```

Comment:
typo: Coverge -> Coverage

Reply:
Fixed in https://github.com/keep-network/coverage-pools/pull/55/commits/e3816ce69b354db0468b9327952b4446a90885bc